model/index.js

```diff
@@ -1,8 +1,17 @@
 const db = require('../db/index.js');
 
 let model = {
-  post: () => {
-    console.log(`you've hit model.post endpoint`);
+  post: (args) => {
+    let query = 'INSERT INTO seller_info (seller_rating, total_sales, seller_name, seller_city, seller_state, on_etsy_since) VALUES ($1, $2, $3, $4, $5, $6)';
```

Comment:
does this work without having an array after the placeholder values of the actual data variable names? E.g. INSERT INTO seller_info (columns) VALUES (value placeholders), [actual_variable name1, actual_variable_name2, etc...]

Reply:
Another example: https://github.com/vitaly-t/pg-promise#index-variables